frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -1,105 +1,141 @@
 # Contributing Data
 
-CELLxGENE supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you have single cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
+CELLxGENE supports a rapidly growing single-cell data corpus because of generous contributions from researchers like you!
 
-## Publishing Process
+## Submission and Publication Process
 
-The process for submission to the portal is:
-
-- You confirm we can support your submission by reaching out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com) with a description of the data that you'd like to contribute.
-- We confirm that we will accept your data.
-- You prepare your data according to our submission [requirements](#dataset-requirements) and send us your files.
-- We upload to a private Collection where you can review.
-- You prepare revised data and send us your revised files, as needed.
-- We publish the data when you tell us to.
+1. Review the [Data Eligibility](#data-eligibility) criteria to ensure your data complies with these requirements
+2. [Contact us] with a description of the data that you'd like to contribute to confirm that we will accept your data
+3. Once confirmed, you send us files prepared according to the submission [Requirements](#dataset-requirements)
+5. We upload to a private Collection where you can review
+6. The submission can be revised, as needed
+7. The data are made openly available when you are ready
 
 ## Dataset Requirements
 
 ### Data Eligibility
 
-CELLxGENE is focused on supporting the global community attempting to create references of human cells and tissues. As a result, there are a few kinds of datasets that we are **not** likely to accept at this time:
+CELLxGENE supports most single-cell RNA-seq and ATAC-seq data, but a few types of data are **not** accepted at this time:
 
 - drug screens
 - cell lines
-- organisms other than mouse or human
-- assays not on the [CELLxGENE Census accepted assays list](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/census_accepted_assays.csv)
-  - These additional assays are pending Census acceptance and will be accepted:
-    - Visium (non-HD)
-    - Slide-seq
-    - Expression measurements of multi-modal assays (e.g. 10x multiome, mCT-seq)
+- species not on the [supported list](https://chanzuckerberg.github.io/single-cell-curation/latest-schema.html#organism_ontology_term_id)
+- assays not on the [supported list](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/census_accepted_assays.csv)
+  - these additional assays are pending Census acceptance and will be accepted:
+    - expression measurements from multi-modal assays (e.g. 10x multiome, mCT-seq)
+    - unpaired ATAC-seq measurements
```

Comment:
"unpaired" is referenced twice. Does it need a definition and/or link to the schema definition?